test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -900,7 +900,7 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 			"spec.unschedulable": "false",
 		}.AsSelector().String()})
 		framework.ExpectNoError(err)
-		gomega.Expect(nodesToBreakCount <= len(nodes.Items)).To(gomega.BeTrue())
+		gomega.Expect(nodesToBreakCount).To(gomega.BeNumerically("<=", len(nodes.Items)))
```

Comment:
Can we find all the gomega code with a similar pattern and change them all? Maybe we could have something like `framework.ExpectNumerically`.

/cc @oomichi

Reply:
It's not clear to me what the benefit of using gomega over:
```
if nodesToBreakCount <= len(nodes.Items) {
  t.Fail
}
```